packages/multimedia/ffmpeg/package.mk

```diff
@@ -34,6 +34,13 @@ PKG_AUTORECONF="no"
 # Dependencies
 get_graphicdrivers
 
+if [ "$V4L2_SUPPORT" = "yes" ]; then
+  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET v4l-utils libdrm"
+  FFMPEG_V4L2="--enable-v4l2_m2m --enable-libv4l2 --enable-libdrm"
+else
+  FFMPEG_V4L2="--disable-v4l2_m2m --disable-libv4l2 --disable-libdrm"
```

Comment:
explicitly disabling libdrm could cause issues later when we also have `FFMPEG_RKMPP` that wants to enabled libdrm

Reply:
@Kwiboo be aware that this is a feature branch. It's still relevant but won't effect master for a while yet.